packages/@pollyjs/adapter-fetch/tests/integration/server-test.js

```diff
@@ -23,6 +23,7 @@ describe('Integration | Server', function() {
     const json = await res.json();
 
     expect(res.status).to.equal(200);
+    expect(res.statusText).to.equal('OK');
```

Comment:
Can you create a new test inside the `adapter-test.js` file instead?

Reply:
I think it's done. Tell me if you need more changes  :)

Reply:
Looks great, ty!